cylc/flow/config_schema.py

```diff
@@ -0,0 +1,660 @@
+        'max active cycle points': [VDR.V_INTEGER, 3],
+        'spawn to max active cycle points': [VDR.V_BOOLEAN],
+        'queues': {
+            'default': {
+                'limit': [VDR.V_INTEGER, 0],
+                'members': [VDR.V_STRING_LIST],
+            },
+            '__MANY__': {
+                'limit': [VDR.V_INTEGER, 0],
+                'members': [VDR.V_STRING_LIST],
+            },
+        },
+        'special tasks': {
+            'clock-trigger': [VDR.V_STRING_LIST],
+            'external-trigger': [VDR.V_STRING_LIST],
+            'clock-expire': [VDR.V_STRING_LIST],
+            'sequential': [VDR.V_STRING_LIST],
+            'exclude at start-up': [VDR.V_STRING_LIST],
+            'include at start-up': [VDR.V_STRING_LIST],
+        },
+        'xtriggers': {
+            '__MANY__': [VDR.V_XTRIGGER],
+        },
+        'graph': {
+            '__MANY__': [VDR.V_STRING],
+        },
+    },
+    'runtime': {
+        '__MANY__': {
+            'inherit': [VDR.V_STRING_LIST],
+            'init-script': [VDR.V_STRING],
+            'env-script': [VDR.V_STRING],
+            'err-script': [VDR.V_STRING],
+            'exit-script': [VDR.V_STRING],
+            'pre-script': [VDR.V_STRING],
+            'script': [VDR.V_STRING],
+            'post-script': [VDR.V_STRING],
+            'extra log files': [VDR.V_STRING_LIST],
+            'work sub-directory': [VDR.V_STRING],
+            'meta': {
+                'title': [VDR.V_STRING, ''],
+                'description': [VDR.V_STRING, ''],
+                'URL': [VDR.V_STRING, ''],
+                '__MANY__': [VDR.V_STRING, ''],
+            },
+            'simulation': {
+                'default run length': [VDR.V_INTERVAL, DurationFloat(10)],
+                'speedup factor': [VDR.V_FLOAT],
+                'time limit buffer': [VDR.V_INTERVAL, DurationFloat(30)],
+                'fail cycle points': [VDR.V_STRING_LIST],
+                'fail try 1 only': [VDR.V_BOOLEAN, True],
+                'disable task event handlers': [VDR.V_BOOLEAN, True],
+            },
+            'environment filter': {
+                'include': [VDR.V_STRING_LIST],
+                'exclude': [VDR.V_STRING_LIST],
+            },
+            'job': {
+                'batch system': [VDR.V_STRING, 'background'],
+                'batch submit command template': [VDR.V_STRING],
+                'execution polling intervals': [VDR.V_INTERVAL_LIST, None],
+                'execution retry delays': [VDR.V_INTERVAL_LIST, None],
+                'execution time limit': [VDR.V_INTERVAL],
+                'submission polling intervals': [
+                    VDR.V_INTERVAL_LIST, None
+                ],
+                'submission retry delays': [VDR.V_INTERVAL_LIST, None],
+                'host': [VDR.V_STRING],
+                'owner': [VDR.V_STRING],
+                'platform': [VDR.V_STRING],
+                'suite definition directory': [VDR.V_STRING],
```

Comment:
Is this still required?

Reply:
Not sure. What was it ever for?

Reply:
The directory containing the `suite.rc` may contain other source files. Rose installs the suite and mirrors everything so it happens that the suite definition directory is also the suite run directory. Cylc continues to have this separation, but since we are going to introduce `rose suite-run` functionality to Cylc, this setting can probably disappear with the change.

Reply:
Yes, drop it.